pkg/gofr/gofr.go

```diff
@@ -310,86 +309,97 @@ func (a *App) initTracer() {
 	traceExporter := a.Config.Get("TRACE_EXPORTER")
 	tracerURL := a.Config.Get("TRACER_URL")
 
-	// deprecated : tracer_host and tracer_port is deprecated and will be removed in upcoming versions
+	// deprecated : tracer_host and tracer_port is deprecated and will be removed in upcoming versions.
 	tracerHost := a.Config.Get("TRACER_HOST")
 	tracerPort := a.Config.GetOrDefault("TRACER_PORT", "9411")
+	authHeader := a.Config.Get("TRACER_AUTH_KEY")
 
-	if tracerURL != "" && traceExporter == "" {
-		a.Logger().Error("missing TRACE_EXPORTER config, should be provided with TRACER_URL to enable tracing")
-		return
-	}
-
-	//nolint:revive // early-return is not possible here, as below is the intentional logging flow
-	if tracerURL == "" && traceExporter != "" && !strings.EqualFold(traceExporter, "gofr") {
-		if tracerHost != "" && tracerPort != "" {
-			a.Logger().Warn("TRACER_HOST and TRACER_PORT are deprecated, use TRACER_URL instead")
-		} else {
-			a.Logger().Error("missing TRACER_URL config, should be provided with TRACE_EXPORTER to enable tracing")
-			return
-		}
+	if tracerURL == "" && tracerHost != "" && tracerPort != "" {
```

Comment:
Shouldn't be this?

```suggestion
	if tracerURL == "" && (tracerHost != "" || tracerPort != "") {
```

Reply:
Right, I would have removed it as we no longer need the default value

Reply:
I can do this easily. Do you need me to do it?

Reply:
My point was to either remove the default value,vor the test on a field that have a default value, so that cannot be empty

Reply:
> I can do this easily. Do you need me to do it?

I only have a phone, so yes, I would o me if you could

Reply:
All done 🌞

Reply:
All done 🌞